.github/workflows/publish-gradle-plugin.yml

```diff
@@ -0,0 +1,21 @@
+name: publish gradle plugin
```

Comment:
I think we also need to publish snapshot versions on every commit into the master. This will help to check the changes before making a release

Reply:
Gradle Plugin Portal doesn't support snapshot versions. We have two ways:
1) test plugin locally.
2) configure snapshot publication to sonatype.

Reply:
Okay, I understand. I would prefer to add snapshots publication (because it helps with using and testing features before the final release) however, I am okay if we stick to the releases only.

But I would add the build on the master branch anyway to fill the Gradle cache with actual data (not from the feature braches)